.github/workflows/nightly-android-testbuild.yml

```diff
@@ -113,34 +116,31 @@ jobs:
           encodedString: ${{ secrets.KEYSTORE_FILE }}        
       # https://github.com/android-actions/setup-android
       - name: Setup Android SDK
-        uses: android-actions/setup-android@v2
+        uses: android-actions/setup-android@v3
       - name: Building webapp
         run: |
           export platform="github-actions-play"
           export serverprefix="testing"
           export VITE_CLUSTER_ADMIN_GROUP_NAME=cluster-admin
-          export VITE_COMMIT_VERSION=$(bin/getDescribeString.sh android)
+          export VITE_COMMIT_VERSION=${{env.version_string}}
           export VITE_SHOW_WIPE=true
-          export VITE_SHOW_NEW_NOTEBOOK=true
           export VITE_SHOW_MINIFAUXTON=true
-          export VITE_DIRECTORY_HOST=db.testing.fieldmark.app
-          export VITE_DIRECTORY_PORT=443
+          export VITE_CONDUCTOR_URL=https://conductor.testing.fieldmark.app
```

Comment:
Okay - noting it would be nice to make use of GitHub's environments feature here to not hardcode into the workflow file. With this approach the downstream forked users can just update their variables for a given environment rather than having to diverge from this workflow file. But okay for now I think.

Reply:
Agree, some of these can be moved to the env, but some are fixed for a production deploy.